new-site/css/style.css

```diff
@@ -245,3 +245,242 @@ textarea {
   }
 }
 
+/************************************************************************************
+MEDIA QUERIES
+*************************************************************************************/
+
+/* logo */
+#logo {
+  width: 100px;
+  height: 70px;
+  float: left;
+  color: white;
+  font-size: 100%;
+}
+
+
+/* navbar */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1.5rem 7%;
```

Comment:
Mixing `rem` and `%` units can get confusing. In general, avoid using `%` in padding and margin.